README.md

```diff
@@ -1,2 +1,102 @@
 # leaderboard-server
-A leaderboard service for Mozilla Geolocation Stumbling
+
+The leaderboard-server is a RESTful service which implements an API
+for submitting data which describes the geolocation stumbling efforts
+of contributors to the Mozilla Location Service project.  Mozilla Stumbler
+users can submit data to the leaderboard-server about how many networks
+were detected within a geospatial 'tile' at a given time, and then query
+the top contributors.
+
+# API Interface
+
+Add Stumbles
+----
+ Submit the number of networks detected within a series of tiles.
+
+* **URL**
+
+  https://leaderboard.services.mozilla.com/backend/add_stumbles
+
+* **Method:**
+
+  `POST`
+
+*  **URL Params**
+
+  None
+
+* **Data Params**
+
+  POST body (JSON encoded)
+
+        {
+          items: [
+            {
+              tile_east: <int>,
```

Comment:
easting is the correct word for projected meters. more accurately we could specify tile_easting_m to indicate it is a meters unit.